types/stripe-checkout/index.d.ts

```diff
@@ -33,4 +33,4 @@ interface StripeCheckoutOptions {
     closed?(): void;
 }
 
-declare var StripeCheckout: StripeCheckoutStatic;
+export var StripeCheckout: StripeCheckoutStatic;
```

Comment:
I don't believe this is correct; see https://unpkg.com/stripe-checkout@1.0.9/index.js; it does `module.exports = Stripe`. There's no exported `StripeCheckout` variable.

Likely this needs to be an `export =` (and then likely all of the namespace shenanigans if you want the types to be importable).